gcs/CONFIGURATION.md

```diff
@@ -503,6 +503,12 @@ permissions (not authorized) to execute these requests.
         streaming requests as soon as first backward read or forward read for
         more than `fs.gs.inputstream.inplace.seek.limit` bytes was detected.
 
+    *   `AUTO_RANDOM` - in this mode connector starts with bounded range
+        requests when reading non gzip-encoded object and switches to streaming
+        request, bounded by `fs.gs.block.size`, if previous two requests follows
+        sequential read pattern i.e. forward seeks which are within
```

Comment:
I think there is scope for improvement in the documentation. I did not get the gist of this flag by reading the documentation.

It is explaining WHAT the feature is doing. If you can also add WHEN this flag makes, sense, it would be useful the future readers of the documentation.

Reply:
Updated